apps/notifications/templates/civic_europe_notifications/emails/notify_followers_winner.en.email

```diff
@@ -2,15 +2,11 @@
 {% load verbose_name %}
 
 {% block subject %}
-"{{ idea.idea_title }}" has won.
-{% endblock %}
-
-{% block headline %}
-The proposal "{{ idea.idea_title }}" is a winning idea.
+The idea "{{ idea.idea_title }}" is a winning idea
```

Comment:
It's actually renamed to title, so it has to be idea.title here and in a few other places